README.md

```diff
@@ -58,4 +103,61 @@ An example of a **meaningful** change:
 - `falsepos-...` files are cases any filter should catch
 - `truepos...` files are cases of changes we care about
 
-Filters will need to be tested (and possibly trained) against real datasets at some point
+This is a small but illustrative sample. Many more samples will be made
+available as soon as possible.
+
+### Schema
+
+This describes the schema of the SQL databases shared by the Rails app in
+web-monitoring-db and the Python processing bakend in web-monitoring-processing.
+Review the Definition of Terms section above, which corresponds to these tables.
+
+Every table includes:
+
+* uuid: UUID4 unique indentifier
+* created_at: internal detail of the database
+* updated_at: internal detail of the database
+
+in addition to the table-specific fields listed below.
+
+#### Pages
+
+* url: URL, which may be updated over time if a page is moved
+* title: `<title>` tag
+* agency: Governemnt agency
+* site: Goverment site (?)
+
+#### Versions
+
+* page_uuid: reference to a Page
+* capture_time: when this snapshot of the Page was acquired
+* uri: path to stored (HTML) data; could be a filepath, S3 bucket, etc.
+* version_hash: sha256 hash of stored data
+* source_type: name of source (such as 'Internet Archive')
+* source_metadata: JSON blob of extra info particular to the source
+
+#### Changes
+
+* uuid_from: reference to the "before" Version
+* uuid_to: reference to the "after" Version
+* priority: a number between 0 and 1 where 1 is high priority
+* current_annotation: for convenience, a reduction of one or more submitted Annotations
```

Comment:
We should probably be clear that this is a JSON object. Would it be helpful to describe here how we “reduce” the annotations? e.g. this comment in the `-db` source:

> Example: the following annotations:
> ```json
> {"a": "one", "b": "two", "c": "three"}
> {"a": "Not one anymore!", "b": null}
> ```
> Result in the following materialized annotation:
> ```json
> {"a": "Not one anymore!", "c": "three"}
> ```